assets/js/components/includes/DropdownFilter.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <select @change="getIndex(items)">
```

Comment:
there was no need to pass items here, the component is well aware of its items.
besides being bad design, you might also manage to override vue's default first argument.